src/tsx/timeline.tsx

```diff
@@ -303,4 +331,16 @@ class Timeline extends React.PureComponent<IProps, IState> {
   }
 }
 
+function getActionText(action: IAction): string {
+  if (action == null) {
```

Comment:
```==``` and ```!=``` are dangerous. Do we know the exact types here, so we can use ```===``` and ```!==```?

Reply:
So it could be either undefined or null in practice? I think I prefer checking for both separately with ```===``` then, just to be explicit (and make it clear in the function signature).

(If it's more of a "just in case", I think we can skip it.)

Reply:
I realized it was more of a "just in case" from the early stages so removed it completely.